handsontable.d.ts

```diff
@@ -76,8 +76,10 @@ declare namespace _Handsontable {
     selectCellByProp(row: number, prop: string, endRow?: number, endProp?: string, scrollToCell?: boolean): boolean;
     setCellMeta(row: number, col: number, key: string, val: string): void;
     setCellMetaObject(row: number, col: number, prop: object): void;
-    setDataAtCell(row: number | any[], col: number, value: string | object, source?: string): void;
-    setDataAtRowProp(row: number | any[], prop: string, value: string, source?: string): void;
+    setDataAtCell(row: number, col: number, value: string | object, source?: string): void;
+    setDataAtCell(row: [number, number, string | object][], source?: string): void;
```

Comment:
`row` seems the wrong name, but I believe it matches the documentation this way.